### DISCUSSION TITLE
Audio message support

### DISCUSSION THREAD

**Discussion by lucasjinreal** [flyerhq/flutter_chat_ui#149] [Ideas & Requests]
Does there any discussion abou this feautre? The bottom bar, add a voice sending

**Comment by demchenkoalex** [comment]
There is an old PR if you want to have some guidance. It was never merged because the sound library is not the best, and creating a new one for mobile, web and desktop is even larger than this project.

**Comment by lucasjinreal** [comment]
@demchenkoalex the sound indeed annoying. A best sound library exactly a basic requirements for such a beautiful lib.

It is pity that whole community can not even found a satisfacying lib for sound.

**Comment by lucasjinreal** [comment]
@demchenkoalex it seems there are some enhancement fork of original widely used sound lib: https://pub.dev/packages/tau_sound

We can discuss here on how to make such a feature based on current resources we have right now. Because if there were no a complete package for sound, that will probably not have except based upon current implementations.

**Reply by demchenkoalex** [reply]
GPL license again.. They suggest a branch if we don't want to use GPL, but I can't add dependencies from branches to the package that is being distributed through the pub.dev.

**Reply by demchenkoalex** [reply]
Ideally we would found something MIT or Apache 2.0 licensed, but last time I checked there were only libs for playing the sound and not recording it.
